addons/common/functions/fnc_actionKeyPressed.sqf

```diff
@@ -0,0 +1,48 @@
+#include "..\script_component.hpp"
+#include "\a3\ui_f\hpp\defineDIKCodes.inc"  
+/*
+ * Author: PabstMirror
+ * Checks if actionName is being pressed (from ui "KeyDown" event data)
+ * Limitations: Does not check joysticks, different left/right modifier keys, double tap binds
+ *
+ * Arguments:
+ * 0: Action Name <STRING>
+ * 1: Key <NUMBER>
+ * 2: Shift <BOOL>
+ * 3: Ctrl <BOOL>
+ * 4: Alt <BOOL>
+ *
+ * Return Value:
+ * <BOOL>
+ *
+ * Example:
+ * ["Watch", 24, false, false, false] call ace_common_fnc_actionKeyPressed
+ * ["TeamSwitch", 22, false, true, false] call ace_common_fnc_actionKeyPressed
+ *
+ * Public: No
+ */
+
+ params [["_actionName", "", [""]], ["_key", -1, [0]], ["_shift", false, [false]], ["_ctrl", false, [false]], ["_alt", false, [false]]];
+
+private _return = false;
+private _actionKeys = actionKeysEx _actionName;
+{
+    _x params [["_mainKeyArray", []], ["_comboKeyArray", []], ["_isDoubleTap", false]];
+    _mainKeyArray params ["_mainDik", "_mainDevice"];
+    if (
+        (_mainDik == _key)
+        && {_mainDevice == "KEYBOARD"}
```

Comment:
```suggestion
        && _mainDevice isEqualTo "KEYBOARD"
```
If the correct casing is always returns.

And surely the lazy eval doesn't save anything with such a cheap check.